docs/using-workflows/adding-connections.md

```diff
@@ -22,3 +24,24 @@ To set the required values for the connection, on the workflow's page, expand th
 ## Managing connections 
 Connections can be managed from the Connections menu option on the left nav. You can add new connections, override the values for existing connections, or delete existing connections. 
 
+## About Connections
+Connections are encrypted when entered and remain encrypted until they are used by a workflow.
+
+When workflows are run, workflow actions that require a connection request the connection value from the Relay Interace. Secrets can be accessed from within an action using one of the Relay SDKs:
```

Comment:
When you run a workflow, actions that need a connection request it from Relay's secret store. You can access secrets from within an action with one of the Relay SDKs
(typo on Interface)